### ISSUE TITLE
Having problems using walkingbase plugin with screenreader

### ISSUE THREAD

**Issue by denizsincar29** [philxan/WalkingBass#3]
Hello.
MuseScore has improved it's accessibility in the version 4. Blind people can edit and view scores and the screenreader tells everything including notes, bar lines and pannels.
When I open the walking bass plugin, i got a normal score window, but I cannot navigate to the walking bass pannel, cannot open the menubar with Alt key and stuck. I thaught that all plugins are screenreader accessible by default.
Can you provide me a tutorial on using this plugin with a screenreader?
Thank you.
